### PR TITLE
Update template to generate based on payment tax year automatically

### PR DESCRIPTION
**Pull request by 1993jamesgrant**
The template needs updating manually currently, this will general the correct year range based on the payment tax year.

### CODE REVIEW

**Review by asmega — APPROVED** [review]

**Review by rjlynch — APPROVED** [review]

File: app/helpers/payment_mailer_helper.rb [hunk 1]
```diff
@@ -1,4 +1,9 @@
 module PaymentMailerHelper
+  def tax_year_for(date)
+    year = (date.month < 4 || (date.month == 4 && date.day < 6)) ? date.year - 1 : date.year
+    "#{year} to #{year + 1}"
+  end
+
```
Comment (kenfodder) [inline app/helpers/payment_mailer_helper.rb:6]: Maybe add some edge case testing for this, and would we ever pass in `nil` into this method?

I'm not fussed possibly move this as a utility method somewhere, but no need if not used elsewhere.

**Review by kenfodder — APPROVED** [review]
